src/main/org/deidentifier/arx/criteria/EDDifferentialPrivacy.java

```diff
@@ -58,11 +58,11 @@
     /** Parameter */
     private DataSubset               subset;
     /** Parameter */
-    private transient DataManager    manager;
-    /** Parameter */
     private transient boolean        deterministic    = false;
     /** Parameter */
     private DataGeneralizationScheme generalization;
+    /** Indicates if this instance was already initialized */ 

```

Comment:
Comment on why this variable is transient.

Reply:
An instance is initialized when it is used to anonymize a dataset. When this model is used more than once, a new subset needs to be drawn before each use (i.e. when "initialized=true". The field is transient, because we need to preserve the subset, when the model is loaded from a project file (indicated by "initialized=false").